configure.ac

```diff
@@ -533,6 +533,9 @@ AC_CHECK_DECLS([bswap_16, bswap_32, bswap_64],,,
                  #include <byteswap.h>
                  #endif])
 
+AC_MSG_CHECKING(for __builtin_clzl)
```

Comment:
I don't know of any compilers that don't. I'd be happy to write a fallback if there is one that isn't.